devkit/Makefile

```diff
@@ -26,13 +26,13 @@ TAG  ?= $(major).$(minor).$(patch)-$(git_rev)
 IMAGE_REPO_TAG := $(REPO):$(TAG)
 
 DOCKER_VERSION ?= 20.10.14_ce
-GO_VERSION ?= 1.19.10
-GOLANGCI_VERSION ?= 1.48.0
+GO_VERSION ?= 1.21.3
+GOLANGCI_VERSION ?= 1.55.2
```

Comment:
devkit change need to pass the multiple jenkins job,  (csi-build,  csi-kind-acceptance,  csi-ci), as these tests are based on the devkit image.

Reply:
csi-build - PASSEDcsi-kind-acceptance, csi-ci